### ISSUE TITLE
CI: json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by certik** [lfortran/lfortran#3707]
https://github.com/lfortran/lfortran/actions/runs/8416999752/job/23044644284?pr=3704

```
 VERIFY HASH:  ../integration_tests/arrays_op_4.f90
 VERIFY HASH:  ../integration_tests/arrays_op_5.f90
Traceback (most recent call last):
  File "/home/runner/work/lfortran/lfortran/./run_tests.py", line 496, in <module>
    tester_main("LFortran", single_test)
  File "/home/runner/work/lfortran/lfortran/src/libasr/compiler_tester/tester.py", line 478, in tester_main
    for f in futures:
  File "/home/runner/micromamba/envs/lf/lib/python3.10/concurrent/futures/_base.py", line 621, in result_iterator
    yield _result_or_cancel(fs.pop())
  File "/home/runner/micromamba/envs/lf/lib/python3.10/concurrent/futures/_base.py", line 319, in _result_or_cancel
    return fut.result(timeout)
  File "/home/runner/micromamba/envs/lf/lib/python3.10/concurrent/futures/_base.py", line 451, in result
    return self.__get_result()
  File "/home/runner/micromamba/envs/lf/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/home/runner/micromamba/envs/lf/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/runner/work/lfortran/lfortran/./run_tests.py", line 223, in single_test
    run_test(
  File "/home/runner/work/lfortran/lfortran/src/libasr/compiler_tester/tester.py", line 317, in run_test
    do = json.load(open(jo))
  File "/home/runner/micromamba/envs/lf/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/home/runner/micromamba/envs/lf/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/runner/micromamba/envs/lf/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/runner/micromamba/envs/lf/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by Shaikh-Ubaid** [comment]
Seems like another false failure. Noted it here https://github.com/lfortran/lfortran/issues/3619..

**Comment by certik** [comment]
Fixed by #4150.
